phira/locales/en-US/settings.ftl

```diff
@@ -3,61 +3,64 @@ label = SETTINGS
 
 general = General
 audio = Audio
-chart = Beatmap
+chart = Chart
 debug = Debug
 about = Info
 
 item-lang = Language
-item-offline = Offline mode
-item-offline-sub = You can't upload playing record in offline mode
-item-server-status = Server status
-item-server-status-sub = Navigate to the webpage to check the server status
-check-status = Check
+item-offline = Offline Mode
+item-offline-sub = Disables all online functionality.
```

Comment:
First word (a verb) of a sentence in imperative mood should use base form. Same as others